numpy/core/src/umath/fast_loop_macros.h

```diff
@@ -82,6 +84,29 @@
                                    steps[2] == sizeof(tout))
 
 
+/*
+ * loop with contiguous specialization
+ * op should be the code storing the result in `tout * out`
+ * combine with NPY_GCC_OPT_3 to allow autovectorization
+ * should only be used where its worthwhile to avoid code bloat
+ */
+#define BASE_OUTPUT_LOOP(tout, op) \
+    OUTPUT_LOOP { \
+        tout * out = (tout *)op1; \
+        op; \
+    }
+#define OUTPUT_LOOP_FAST(tout, op) \
+    do { \
+    /* condition allows compiler to optimize the generic macro */ \
+    if (IS_OUTPUT_CONT(tout)) { \
```

Comment:
Should be indented.

Reply:
I think this matches the existing macros, sadly

Reply:
Please see https://github.com/numpy/numpy/pull/13208 - I have fixed the indentation for all macros in this file